docs/user/Build.md

```diff
@@ -74,53 +74,47 @@ it can be uploaded in the "Relevant log output" section of OpenROAD
 
 You may follow our helper script to install dependencies as follows:
 ``` shell
-sudo ./etc/DependencyInstaller.sh
+sudo ./etc/DependencyInstaller.sh -base
+./etc/DependencyInstaller.sh -common -local
 ```
 
 ```{warning}
-`etc/DependencyInstaller.sh` defaults to installing system 
-packages and requires sudo access. These packages can affect
-your environment.
+`sudo ./etc/DependencyInstaller.sh [-all|-common]` defaults to
+installing packages on /usr/local.
+To avoid this bahavior use -local flag or -prefix <PATH> argument.
 ```
 
 ### Compiling OpenROAD Manually
 
 ``` shell
-mkdir build && cd build
-cmake ..
-make
-sudo make install 
+./etc/Build.sh
+sudo make install
```

Comment:
I would leave out all instructions in the manual to install OpenROAD into /usr/local, this generally harms the system and should be discouraged.

Installing into /usr/local is really, to my mind, a use-case for Docker only.

Reply:
I will split this into a separate section since this `make install` only targets the `openroad` bin. Installing `openroad` at the system level has helped when running tutorials or allowing sysadmins to maintain one bin for all users of a shared machine.

Reply:
It is also helpful if you maintain many copies of OR and don't have to update the dependencies for each.  It can be a non-default option but it should be an option.

Reply:
Yep. No sucker punches, but if you ask for it explicitly, then you will get it and you deserve the result.